app/javascript/lib/shared/modules/vue-filters.js

```diff
@@ -27,10 +27,15 @@ export const date = (value, opts = {}) => {
 }
 
 export const truncate = (value, opts = {}) => {
-  const omission = opts['omission'] || '...';
   const length = opts['length'] || 30;
+  const str = `${value || ''}`
```

Comment:
Iterating over your commons. That's the way! 🥳

Reply:
I don't understand what you meant, is it ok then 😅 ?

Reply:
Yes, it's a compliment. :)